Dockerfile

```diff
@@ -31,7 +33,17 @@ RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /et
     && rm -rf /tmp/* /var/tmp/* \
     && groupmod -g 1000 users \
     && useradd -u 911 -U -d /config -s /bin/false abc \
-    && usermod -G users abc
+    && usermod -G users abc \
+    && addgroup -S privoxy && adduser -S privoxy -G privoxy \
+    && mkdir /opt/privoxy \
+    && curl -sL https://www.privoxy.org/sf-download-mirror/Sources/3.0.29%20%28stable%29/privoxy-3.0.29-stable-src.tar.gz | tar -C /opt/privoxy --strip-components=2 -xz \
+    # && git clone https://www.privoxy.org/git/privoxy.git /opt/privoxy \
+    && cd /opt/privoxy \
+    && autoheader \
+    && autoconf \
+    && ./configure --enable-compression --with-brotli  --with-mbedtls --enable-extended-statistics  \
+    && make -j4 \
+    && make install
```

Comment:
`make install-strip` will also strip symbols from the binary, making it a bit smaller.

Ref: https://www.privoxy.org/gitweb/?p=privoxy.git;a=blob;f=GNUmakefile.in;h=fcd44c4f449897df249686c034466a1da031e215;hb=HEAD#l841

Reply:
3.0.29 is intentional :) I found later releases crash after a few hours but 3.0.29 does not have this issue :)

Reply:
`install-strip` I have added, thanks for the great idea :) Once Privoxy becomes stable past 3.0.29 again, I will implement your XML parser 👍